engine/index.js

```diff
@@ -298,6 +299,49 @@ function populateCanIUsePercent(canIUseData, features) {
   });
 }
 
+const storedFields = ['firefox_status', 'spec_status', 'opera_status',
+                      'webkit_status', 'ie_status'];
+function checkForNewData(features) {
+  const client = redis.createClient(process.env.REDISCLOUD_URL, { no_ready_check: true });
```

Comment:
We've got a process which stops after generating a report instead of running as a server. I tried to do it the way we do with express and tests are hanging waiting for the connection to close.